crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -611,6 +614,65 @@ impl Mesh {
         }
     }
 
+    /// Calculates the [`Mesh::ATTRIBUTE_NORMAL`] of an indexed mesh, smoothing normals for shared
+    /// vertices.
+    ///
+    /// # Panics
+    /// Panics if [`Mesh::ATTRIBUTE_POSITION`] is not of type `float3`.
+    /// Panics if the mesh has any other topology than [`PrimitiveTopology::TriangleList`].
+    /// Panics if the mesh does not have indices defined.
+    ///
+    /// FIXME: The should handle more cases since this is called as a part of gltf
```

Comment:
```suggestion
    /// FIXME: This should handle more cases since this is called as a part of gltf
```

Reply:
Good catch, I think this typo is in the existing set of comments as well. Can fix there.